delfin/common/constants.py

```diff
@@ -29,8 +29,9 @@ class StorageStatus(object):
     NORMAL = 'normal'
     OFFLINE = 'offline'
     ABNORMAL = 'abnormal'
+    DEGRADED = 'degraded'
 
-    ALL = (NORMAL, OFFLINE, ABNORMAL)
+    ALL = (NORMAL, OFFLINE, ABNORMAL, DEGRADED)
```

Comment:
After adding this, where would use it? Would it in another PR?

Reply:
Storage devices such as NetApp and 3PAR have the status value degraded. This enumeration can be added to more accurately describe the device status.